src/test/java/edu/harvard/iq/dataverse/api/UtilIT.java

```diff
@@ -372,7 +372,16 @@ static Response showDataverseContents(String alias, String apiToken) {
                 .header(API_TOKEN_HTTP_HEADER, apiToken)
                 .when().get("/api/dataverses/" + alias + "/contents");
     }
-    
+
+    static Response getGuestbookResponses(String dataverseAlias, Long guestbookId, String apiToken) {
+        RequestSpecification requestSpec = given()
+                .auth().basic(apiToken, EMPTY_STRING);
```

Comment:
```suggestion
                .header(API_TOKEN_HTTP_HEADER, apiToken);
```

Reply:
Seriously?! I've been stuck on this for a long time.

Reply:
I guess I should not have started from a SWORD call 😞